sdk/core/core/inc/az_result.h

```diff
@@ -61,14 +61,16 @@ typedef enum az_result
 {
   AZ_OK = _az_RESULT_MAKE_SUCCESS(AZ_FACILITY_CORE, 0),
   AZ_CONTINUE = _az_RESULT_MAKE_SUCCESS(AZ_FACILITY_CORE, 1),
+  AZ_CANCELED = _az_RESULT_MAKE_ERROR(AZ_FACILITY_CORE, 0),
```

Comment:
Add a comment to break up the errors from the successes
//Success results
AZ_OK
AZ_CONTINUE
//Error results
AZ_CANCELLED

Reply:
Even more, I think it's important we document what AZ_OK vs AZ_CONTINUE mean.

Reply:
Break up is done.

Regarding documenting AZ_CONTINUE:
Honestly, I am not fully sure what AZ_CONTINUE means, but none of the result codes are documented at this moment. Let's document all of them in the scope of another PR.

I am going to resolve this conversation and I consider feedback as 'done'. If you disagree and think I should cover it in the scope of this PR, please reopen the conversation or ping me.